### PR TITLE
TC-143 break even landing page

### PR DESCRIPTION
**Pull request by kscoulter**
- Create a landing page for break even calc http://localhost:3000/breakevencalculator
- Add info section
- Add blue background promo section

Screen Shot 2020-06-08 at 3 51 10 PM

Screen Shot 2020-06-16 at 4 47 29 PM

### CODE REVIEW

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Button } from 'atoms'
+
+import styles from './break-even-calc-landing-page.scss'
+
+class BreakEvenCalculatorPage extends React.Component {
+  calloutContent = {
+    title: "What you'll need to get started",
+    text:
+      'This analysis will help you easily prepare an estimate and visual to include in your business plan. We’ll do the math and all you will need is an idea of the  following information.'
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:10]: remove the extra space "in the  following information".

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            level of production at which the costs of production equal the revenues for a product.
+          </h5>
+          <h3>An important calculation for your business plan</h3>
+          <p>
+            Potential investors in a business not only want to know the return to expect on their
+            investments but also the point when they will realize this return. This is because some
+            companies may take years before turning a profit, often losing money in the first few months or
+            years before breaking even. For this reason, break-even points are an important part of any
+            business plan presented to a potential investor. It can also be a useful tool in future planning
+            for existing businesses to prove their potential turn around after disaster scenarios.{' '}
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:29]: What does this do ? {' '}

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          <div className={styles.infoContainer}>
+            <div className={styles.row}>
+              <div className={styles.calcImage}>
+                <img src="/assets/images/break-even-calculator/Calculator_icon.svg" alt="calculator icon" />
+              </div>
+              <div className={styles.calcText}>
+                Get started with the
+                <h3>Break-even point analysis tool</h3>
+                <p>
+                  Create your break-even analysis with this calculator and determine your business’s
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:42]: for consistency did we want to use business’s or business'

File: src/client/components/routes.jsx [hunk 4]
```diff
@@ -1,7 +1,5 @@
 import Async from 'react-code-splitting'
-
 import React from 'react'
-import { isEmpty } from 'lodash'
```
Comment (peggles2) [inline src/client/components/routes.jsx:3]: are we sure this wont impact any other applications to remove that?

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Button } from 'atoms'
+
+import styles from './break-even-calc-landing-page.scss'
+
+class BreakEvenCalculatorPage extends React.Component {
+  calloutContent = {
+    title: "What you'll need to get started",
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:8]: why do we use double quotes here and single quotes on line 10?

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx (hunk 3 above)
Comment (kscoulter) [reply src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:42]: Not sure, would need to hear something from content team, I guess

File: src/client/components/routes.jsx (hunk 4 above)
Comment (kscoulter) [reply src/client/components/routes.jsx:3]: yes, it's not being used in this file

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx (hunk 3 above)
Comment (peggles2) [reply src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:42]: I got confirmation from Marisa. Change them all to business’s

**Comment by peggles2** [comment]
I put a hold label on this because this is just the landing page and we have to build out the rest. We do not want to merge to master just yet.

**Pull request merged**